Gruntfile.coffee

```diff
@@ -1240,6 +1240,16 @@ module.exports = (grunt) ->
 					"theme/**/*.js"
 					"tasks/*.js"
 				]
+			travis:
+				options:
+					configFile: '.travis.eslintrc.json'
+					quiet: true
+				all:
+					src: [
+						"src/**/*.js"
+						"theme/**/*.js"
+						"tasks/*.js"
+					]
```

Comment:
One final step would be to remove that block altogether and replace the whole task with the following code:

```coffee
eslint:
	options:
		rules: do ->
  			rules = grunt.file.readJSON ".eslintrc.json"
			if process.env.CI
				rules["linebreak-style"] = 2
			rules
		quiet: true
	all:
		src: [
			"src/**/*.js"
			"theme/**/*.js"
			"tasks/*.js"
]
```